frontend/src/services/supabase/client.ts

```diff
@@ -11,3 +12,8 @@ export const client = createClient(
   process.env.REACT_APP_SUPABASE_URL as string,
   process.env.REACT_APP_SUPABASE_ANON_KEY as string
 );
+
+export const clientDatabase = createClient<Database>(
```

Comment:
Why did you create this function? You can use `client` on line 11.

Reply:
the Type was different so i created.

Reply:
Type was different so i created.

Reply:
Type was different so i created.

Reply:
Got it. I changed it.